Rebuild-Black-Business/RBB-Website
# Feature/business result card

# Describe your PR

Adds styles based on the design for the ResultCards on the businesses page. Also adds fallback image support and modal functionality.

## Pages/Interfaces that will change

/businesses

### Screenshots / video of changes

Before: https://gyazo.com/1b80092a8be9773760e9add101f23822
After: https://gyazo.com/f27e85e2d3a1e6492bce071d55ffb6e7

## Steps to test

1. Navigate to the /businesses page
2. Ensure the images appear, or fallbacks are present based on category
3. Ensure report / update modals work when the links are clicked.
